knife/lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -86,8 +86,8 @@ def config_content
             client_rb << "# Using default node name (fqdn)\n"
           end
 
-          if config[:config_log_level]
-            client_rb << %Q{log_level :#{config[:config_log_level]}\n}
+          if chef_config[:log_level]
+            client_rb << %Q{log_level :#{chef_config[:log_level]}\n}
           else
             client_rb << "log_level        :auto\n"
           end
```

Comment:
This should probably be changed to be the same as the linux/unix code in boostrap_context.rb:

```
          unless chef_config[:config_log_level].nil? || chef_config[:config_log_level].empty?
            client_rb << %Q{log_level   :#{chef_config[:config_log_level]}\n}
          end
```

Or else both should be made consistent one way or another.

And it looks like `chef_config[:config_log_level]` should be used instead of `chef_config[:log_level]` unless that was deliberate (in which case the boostrap_context.rb should also be changed.

Reply:
@lamont-granquist We have made the changes as per the comment. looks like chef_config[:config_log_level] is working fine in both cases. Thanks